### ISSUE TITLE
Buttons

### ISSUE THREAD

**Issue by StreiffBerenice** [Orange-OpenSource/Orange-Boosted-Bootstrap#160]
@liyokuna Some colour contrasts are not sufficient. All depends of Orange's Brand but you shouldn't use white on orange -> replace by black text on orange background. See buttons components : https://boosted.orange.com/docs/4.3/boostwatch/#buttons

**Comment by ffoodd** [comment]
This is unfortunately not allowed by Orange digital guidelines for now, however we plan to add a contrast enforcemtn widget with #209 — which would help dealing with invalid contrast.

Be sure we won't forget this one :)
